docs/pip/compile.md

```diff
@@ -60,6 +60,18 @@ $ uv pip compile pyproject.toml --all-extras
 
 Note extras are not supported with the `requirements.in` format.
 
+To lock dependency groups in the current project directory's pyproject.toml:
+
+```console
+$ uv pip compile --group foo
+```
```

Comment:
"To lock a dependency group... For example, to lock the group `foo`"?

We also use ` for `pyproject.toml`.